srcpkgs/gucharmap/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gucharmap'
 pkgname=gucharmap
-version=9.0.4
+version=11.0.2
 revision=1
 build_style=gnu-configure
 configure_args="--disable-schemas-compile  $(vopt_enable gir introspection)
```

Comment:
@Chocimier my OCD says no two spaces after `--dsiable-schemas-compile`.

Reply:
While I agree that removing one of those spaces will be significant improvment to the package, it is not related to update. Please make another pull request.